agent/engine/task_manager.go

```diff
@@ -445,10 +445,12 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 	containerKnownStatus := container.GetKnownStatus()
 
 	eventLogFields := logger.Fields{
-		field.TaskID:    mtask.GetID(),
-		field.Container: container.Name,
-		field.RuntimeID: runtimeID,
-		field.Status:    event.Status.String(),
+		field.TaskID:        mtask.GetID(),
+		field.Container:     container.Name,
+		field.RuntimeID:     runtimeID,
+		"changeEventStatus": event.Status.String(),
```

Comment:
Is there a rule-of-thumb for when logged fields get a constant in the `field` package?

Reply:
No particular rules here, I guess just whenever there is a field being used across multiple files/packages we should probably add it to the constants. In this case I don't think this field or the ones below will ever be used outside this particular function so I didn't bother adding constants.